libcob/ChangeLog

```diff
@@ -1,3 +1,7 @@
+2024-12-11  Emilien Lemaire <emilien.lemaire@ocamlpro.com>
+
+	* fileio.c: Always use `bdb_app_data` global variable to save the
+	pointer to the collating sequence function
```

Comment:
format: please start with a new line (reduces merge conflicts, especially with svn) and indent the second+ line of an entry using two spaces